projecttacoma/node-fhir-server-core
# GitHub actions

# Summary
Migrate the ci build from travis to github actions
## New behavior
The behavior is largely unchanged except you will now see the results from a build at the bottom of the pull requests. ( I believe travis was running for Aysmmetrik before the fork)
## Code changes
- travis.yml - I deleted this file since we don't use it anymore 
- ci.yml - Created a new workflow file that clears the yarn cache, installs the dependencies and runs the unit tests  that are part of the package 
# Testing guidance
To test locally:
Run `yarn cache clean`  to clear out your current set up followed by: `yarn install --frozen-lockfile` and `yarn bootstrap`
And finally run `yarn lerna run test` this should run the unit tests and they should all pass.

To run on github:  Navigate here: https://github.com/projecttacoma/node-fhir-server-core/actions and select re-run jobs.
